neuro_evolution_ctrnn/train.py

```diff
@@ -39,7 +42,7 @@ def parse_args(args=None):
     args = parse_args()
     experiment = Experiment(configuration=config_from_file(args.configuration), result_path=args.result_path,
                             from_checkpoint=args.from_checkpoint, processing_framework=args.processing_framework,
-                            number_of_workers=args.num_workers)
+                            number_of_workers=args.num_workers, checkpoint_to_result=args.checkpoint_to_result)
```

Comment:
Wäre es besser wenn `checkpoint_to_result` in `write_final_checkpoint` umbenannt wird, weil wenn der Parameter gesetzt ist wird doch der finale Checkpoint gespeichert und dann in das Directory geschrieben.

Oder benutzt du das um Checkpoints sozusagen "auszulesen" und zu speichern?

Reply:
> Wäre es besser wenn checkpoint_to_result in write_final_checkpoint umbenannt wird

Ja